translator/reader/read_geometry.cpp

```diff
@@ -362,14 +362,14 @@ void UsdArnoldReadMesh::Read(const UsdPrim &prim, UsdArnoldReaderContext &contex
 class CurvesPrimvarsRemapper : public PrimvarsRemapper
 {
 public:
-    CurvesPrimvarsRemapper(bool remapValues, ArnoldUsdCurvesData &curvesData) : 
+    CurvesPrimvarsRemapper(bool remapValues, bool pinnedCurve, ArnoldUsdCurvesData &curvesData) : 
```

Comment:
We need to set the member var `_pinnedCurve(pinnedCurve), `

Reply:
indeed, test_0184 was failing and after the proposed change it works again